packages/microcosm/src/history.js

```diff
@@ -134,7 +134,7 @@ class History extends Emitter {
 
         // Ignore certain lifecycle actions that are only for
         // internal purposes
-        if (next && (next.command === BIRTH || next.command === START)) {
+        if (next && next.command === START) {
```

Comment:
how would the `next` of any action be `START`?

Reply:
Very good question.